emData/download.sh

```diff
@@ -69,6 +69,9 @@ declare -a processing_modules=(
 
   # Tracklet Processor
   "TP_L1L2D" 
+
+  # Purge Duplicates (TrackMerger)
+  "TM"
```

Comment:
This actually needs to be `"PD"`. In the emulation, which we use to generate test-vectors, the TrackMerger module is called PurgeDuplicates. At some point, we should update the name in the emulation, but for now, you just have to use "PD" when referring to test-vectors.

Once you change this and rerun this script (after running clean.sh in the same directory), you should find an emData/PD/PD/ directory that actually has symbolic links in it.